src/components/TokenClaimDialog.tsx

```diff
@@ -229,12 +238,13 @@ export default function TokenClaimDialog({
               </Trans>
             </Typography>
           )}
-          {!IS_VESDL_LIVE && chainId === ChainId.MAINNET
+          {!gaugesAreActive && chainId === ChainId.MAINNET
```

Comment:
I think you can get rid of `chainId === ChainId.MAINNET` since `gaugeAreActive` already took care of that